aspnetcore/fundamentals/configuration/index.md

```diff
@@ -83,6 +83,8 @@ Using the [default](#default) configuration, the *appsettings.json* and *appsett
 
 See [JSON configuration provider](#jcp) in this document for information on adding additional JSON configuration files.
 
+[!INCLUDE[](~/includes/combine-di.md)]
```

Comment:
To me, this doesn't fit here. The sample code shows configuration and options, but it's really about the extension method wrappers.

Reply:
My thinking was if you're creating a bunch of config services you'd like to know how to clean up startup and move config services.